app/database/scripts/wellsearch/wells_replication_stored_functions.sql

```diff
@@ -820,7 +820,40 @@ COMMENT ON FUNCTION migrate_drilling_methods () IS 'Load Well Drilling Method re
 
 
 -- DESCRIPTION
---  Define the SQL INSERT command that copies the drilling method data from the legacy
+--  Define the SQL INSERT command that copies the water quality data from the legacy
+--  database to the GWELLS table (water_quality_characteristics)
+--
+-- PARAMETERS
+--   None
+--
+-- RETURNS
+--  None as this is a stored procedure
+--
+CREATE OR REPLACE FUNCTION migrate_water_quality_characteristics() RETURNS void as $$
+DECLARE
+  row_count integer;
+BEGIN
+  raise notice '...importing water_quality_characteristic';
+  INSERT INTO water_quality_characteristic(
```

Comment:
water_quality_characteristic is a code table - so I don't think this will work. See line 109, app/backend/wells/models.py. 
There is a link table, called well_water_quality which I think is what you're intending to populate.